eox_tenant/models.py

```diff
@@ -108,6 +108,29 @@ def get_value_for_org(cls, org, val_name):
 
         return None
 
+    @classmethod
+    def get_microsite_for_values(cls, search_term):
+        """
+        Returns a list of microsites searching by the values field.
+        """
+        if search_term == '':
+            return cls.objects.none()  # pylint: disable=no-member
+
+        sql = """
+            SELECT id FROM {table}
+            WHERE JSON_EXTRACT(`values`,"$.*") LIKE %s""" \
+            .format(table=cls._meta.db_table)  # pylint: disable=no-member
+        with connection.cursor() as cursor:
+            cursor.execute(sql, ['%' + search_term + '%'])
+            rows = cursor.fetchall()
+        rows = [row[0] for row in rows]
```

Comment:
@felipemontoya  Q: Can you do something like https://stackoverflow.com/questions/37205793/django-values-list-vs-values to have a flat list from the query?

R: I didn't find something similar for a RAW query